uberfire-project/uberfire-project-api/src/main/java/org/guvnor/common/services/project/service/WorkspaceProjectService.java

```diff
@@ -70,5 +70,9 @@ String createFreshProjectName(final OrganizationalUnit organizationalUnit,
 
     WorkspaceProject resolveProject(final Space space, final String name);
 
+    WorkspaceProject resolveProject(final Space space,
```

Comment:
A PR for `kie-wb-common` will be necessary to fix the compilation of `MockWorkspaceProjectService.java`, which implements this interface.

Reply:
done